src/Pages/LogIn/LoginScreen.js

```diff
@@ -0,0 +1,119 @@
+import React, { Component } from "react";
+import { withRouter } from "react-router-dom";
+import LoginTooltipIcon from "./LoginTooltipIcon";
+import LoginPageLoginBtn from "./LoginPageLoginBtn";
+import CreateAccountBtn from "./CreateAccountBtn";
+import LineContainer from "./LineContainer";
+import API from "../../config";
+
+class LoginScreen extends Component {
+  constructor() {
+    super();
+    this.state = {
+      email: "",
+      password: "",
+      inputType: true,
+      visible: true,
+    };
+  }
+
+  userInfo = (e) => {
+    const { name, value } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  handlePassword = () => {
+    const { inputType, visible } = this.state;
+    this.setState({
+      inputType: !inputType,
+      visible: !visible,
+    });
+  };
+
+  tryLogin = (e) => {
+    const { email, password } = this.state;
+    e.preventDefault();
+
+    if (!email) return alert("유효한 이메일 주소를 입력해야 합니다.");
+    if (!email.includes("@"))
+      return alert("이메일 주소에 '@'를 포함해 주세요.");
+    if (!password) return alert("비밀번호가 필요합니다.");
+    if (
+      !(
+        password.length >= 8 &&
+        password.match(/^(?=.*[0-9])(?=.*[a-zA-Z])([a-zA-Z0-9]+)$/)
+      )
+    ) {
+      return alert(
+        "최소 8자 이상의 유효한 비밀번호를 입력하시기 바랍니다. 비밀번호에는 최소 1개의 숫자와 1개의 문자가 포함되어야 합니다."
+      );
+    } else {
+      fetch(`${API}/user/signin`, {
+        method: "POST",
+        body: JSON.stringify({
+          email: email,
+          password: password,
+        }),
```

Comment:
깨알 팁이지만, 아래와 같이 변경할 수 있습니다 ㅎㅎ
```suggestion
        body: JSON.stringify({
          email,
          password,
        }),
```

Reply:
우와.... 꿀팁bbb
그런데 destructuring 을 하지 않았다면
```
  body: JSON.stringify({
          email: this.state.email,
          password: this.state.password,
        }),
```
였을 것입니다. 
혹시 이렇게 key 와 value 가 동일하지 않아도 가능한 것인지,

아니면 destructuring 을해서 key와 value 가 동일해 져서 가능한 것인지 궁금함이 생기네요!

Reply:
@chan-97 객체의 key와 value의 변수명이 같으면 객체 단축 속성명을 활용해 위 코드처럼 간단히 적어줄 수 있습니다!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Object_initializer

Reply:
최종적으로 사용되는 객체의 key 와 value의 변수명이 같이면 단축 속성명 사용이 가능하다!!
감사합니다😁